### DISCUSSION TITLE
Verbose output in dashboard

### DISCUSSION THREAD

**Discussion by DannyBen** [datreeio/datree#606] [Q&A]
I love how the CLI runs make the results available in the web dashboard. However, I am unable to figure out how to do certain things that I expected to be available in the dashboard:

1. The results on the dashboard do not show any additional information (`--verbose`), as to the exact metadata information, and How to fix
2. I could not find a way to show only failing checks

Also, I am unsure what is the way to share the dashboard results with others - I was hoping to see an "Invite" or "Share" link so I can share with our DevOps team.

**Comment by eyarz** [comment]
@DannyBen the dashboard is the GUI for the centralized policy management. Every change in the dashboard will automatically be propagated to the CLI.

1. "the results on the dashboard" - do you refer to the results on the [history page](https://app.datree.io/cli/invocations)?
2. do you mean you want to filter only failing checks on the history page?

You're right. You can't share the dashboard with your teammates, but you can share your policies via the [token management](https://hub.datree.io/setup/account-token) feature. 
User management (Role-based Access Control) is **currently in development** and will be supported soon.

**Comment by DannyBen** [comment]
Well. Yes.

As I understand it so far - from a "Newly Onboarded User" perspective - the dashboard is currently designed to do two things:

1. Configure policies - either through the interface, or through downloadable config template
2. Present the recent scans in a convenient (and hopefully sharable-) way.

Now, the dashboard does its first job quite well. I was able to configure my policies with zero friction. This aspect exceeded my expectation.
As for the History tab - this aspect falls short in my view. Although it is pleasant to look at, it is not useful at its current state, mainly due to the two reasons I mentioned: 

1. It provides LESS information than the CLI, and the missing information is important.
2. It is not searchable / filterable / sortable.

So basically, it just serves as a "log" of what happened, instead of being an interface for the results.

If this is your intention, than great. But, it seems like it can be converted - with not much effort I presume - to a UI for the found issues. For this, it would need:

1. To have filters / sorting / search
2. To have as much information as possible, at least the same as `--verbose` in the CLI (specific meta information in question + how to fix)
3. Ideally, to be sharable and printable (print media CSS)
4. Ideally, to not just show a "console output" of the CLI - but rather a more structured output, including links to the policy settings of each found issue (so I can easily disable it from a link in the history view, for example).

Again - all these are not criticism, only suggestions and impressions of what I expected.

As a side note - I *do* like how easy it was to create the connection between the CLI and the web dashboard, through the `client_id` and `token` in `~/.datree/config.yaml`.
